ocs_ci/ocs/openshift_lightspeed.py

```
@@ -0,0 +1,565 @@
+            url,
+            headers=self._headers(),
+            verify=self.verify_tls,
+            timeout=self._REQUEST_TIMEOUT,
+        )
+        if response.status_code == 200:
+            logger.info("OLS authorization check passed")
+            return True
+        logger.warning(
+            f"OLS authorization check failed: {response.status_code} {response.text}"
+        )
+        response.raise_for_status()
```

Ruff (0.16.3)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)